libraries/AP_HAL_Linux/Util_RPI.h

```diff
@@ -4,6 +4,14 @@
 
 namespace Linux {
 
+enum LINUX_BOARD_TYPE {
```

Comment:
```suggestion
enum class BoardType {
```

This should really be `enum class`  This is a lot of namespace-polution otherwise.

Follow-up-PR would be nice :-)